packages/lib/src/__tests__/table-generator.test.ts

```diff
@@ -49,50 +49,45 @@ function stripEscapeSequences(str: string): string {
 	return retVal
 }
 
-function rowCount(table: string): number {
-	return table.split('\n').length - 2
-}
+/**
+ * Clean up string created using backticks. Lines are stripped of ` +|` at the beginning of each
+ * line (allowing indentation) and `|` at the end (allowing for spaces at the end of the line
+ * that are not automatically removed by tooling). If there is a new-line at the beginning of the
+ * string, it is also removed.
+ */
+const fixIndent = (input: string): string => input.replace(/^\s*\|/gm, '').replace(/\|$/gm, '').replace(/^\n/, '')
```

Comment:
It is not clear to me why I would need this if I was adding tests here. Is there something preventing us from formatting the expected strings without a function?

Reply:
It just makes the tests look nicer. We can use the mutli-line backtick quoting and still indent and not worry about the editor stripping the whitespace from the end of the line.